### ISSUE TITLE
qthelp builder crash with non ascii content

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#232]
When trying to build qthelp with utf-8 files, I get

```
Exception occurred:
  File "/usr/local/lib/python2.6/dist-packages/Sphinx-0.6.2-py2.6.egg/sphinx/builders/qthelp.py", line 175, in build_qhp
    'files': projectfiles})                                                                                             
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2026' in position 2763: ordinal not in range(128)
```

This can be solved by editing 'builders/qthelp.py' and:
- add 'from codecs import open' to the import section
- add "utf-8" as third parameters to the 2 open functions (lines 112 and 163)

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/232
- Originally reported by: Anonymous
- Originally created at: 2009-07-29T12:43:52.747

**Comment by shimizukawa** [comment]
_From Vincent Meurisse on 2009-07-29 11:01:10+00:00_

Add a patch against the current dev version

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2009-08-06 19:14:53+00:00_

Fixed in <<cset b242c6b47c8f36d0ce9261cd1b11c2a029a23436>>, thanks!
